### ISSUE TITLE
PPL and logits display

### ISSUE THREAD

**Issue by gakingk** [AI-Guru/helibrunna#4]
Hello @AI-Guru, is there any built-in way to show a trained xLSTM model's PPL given a prompt  and give the probabilities of the predicted tokens? I am using this framework for academic purposes, and having some data on these things would be great for better visualization. Also, does this project have a bibtex citation file?

Thanks in advance!
